dags/infra/variables.py

```diff
@@ -11,9 +11,19 @@
 
 AWS_DEFAULT_REGION = Variable.get("aws_default_region", "us-west-2")
 SECRET_AWS_NAME = Variable.get("secret_aws_name", "indexing-aws-creds-prod")
-DB_DUMP_S3_ROLE = Variable.get("db_dump_s3_role", "deafrica-prod-af-eks-db-dump-to-s3")
+DB_DUMP_S3_ROLE = Variable.get(
+    "db_dump_s3_role", "deafrica-prod-af-eks-db-dump-to-s3"
+)
 SECRET_DBA_ADMIN_NAME = Variable.get("db_dba_admin_secret", "dba-admin")
-DB_DUMP_S3_BUCKET = Variable.get("db_dump_s3_bucketname", "deafrica-dev-odc-db-dump")
+DB_DUMP_S3_BUCKET = Variable.get(
+    "db_dump_s3_bucketname", "deafrica-dev-odc-db-dump"
+)
 
 SECRET_EXPLORER_WRITER_NAME = Variable.get("explorer_secret", "explorer-db")
 SECRET_OWS_WRITER_NAME = Variable.get("ows_secret", "ows-db")
+
+# Sentinel-2 US connection, this is based on svc-deafrica-prod-eks-s2-data-transfer
+# user which is in deafrica account
+US_CONN_ID = "deafrica-prod-eks-s2-data-transfer"
```

Comment:
this is too vague, in the future this list expands, it will be hard to tracee back

Reply:
also if its a connid can it go to infra/connections.py

Reply:
Added a s2 prefix to make them more specific to sentinel-2.